### ISSUE TITLE
ValueError: 'nrows' must be an integer >=0

### ISSUE THREAD

**Issue by sprinkjm** [jmscslgroup/strym#27]
While doing the below, I received this error when trying to access either speed or steer angle.
```
Reading the CSV file ../../PandaData/2020_09_17/2020-09-17-14-07-51_2T3Y1RFV8KC014025_CAN_Messages.csv
```

Error info below:
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-7-e5e77758aa55> in <module>
      3 for csv in csvlist:
      4     print("\nReading the CSV file {}".format(csv))
----> 5     r = strymread(csvfile=csv)
      6 
      7     if r.success is True:

~/opt/anaconda3/lib/python3.7/site-packages/strym/strymread.py in __init__(self, csvfile, dbcfile, **kwargs)
    272                     output = output.split(' ')
    273                     n_lines = int(output[0])
--> 274                     self.dataframe = pd.read_csv(self.csvfile,dtype={'Time': np.float64,'Bus':np.uint8, 'MessageID': np.uint32, 'Message': str, 'MessageLength': np.uint16}, nrows=n_lines - 2)
    275 
    276                 else:

~/opt/anaconda3/lib/python3.7/site-packages/pandas/io/parsers.py in parser_f(filepath_or_buffer, sep, delimiter, header, names, index_col, usecols, squeeze, prefix, mangle_dupe_cols, dtype, engine, converters, true_values, false_values, skipinitialspace, skiprows, skipfooter, nrows, na_values, keep_default_na, na_filter, verbose, skip_blank_lines, parse_dates, infer_datetime_format, keep_date_col, date_parser, dayfirst, cache_dates, iterator, chunksize, compression, thousands, decimal, lineterminator, quotechar, quoting, doublequote, escapechar, comment, encoding, dialect, error_bad_lines, warn_bad_lines, delim_whitespace, low_memory, memory_map, float_precision)
    674         )
    675 
--> 676         return _read(filepath_or_buffer, kwds)
    677 
    678     parser_f.__name__ = name

~/opt/anaconda3/lib/python3.7/site-packages/pandas/io/parsers.py in _read(filepath_or_buffer, kwds)
    452 
    453     try:
--> 454         data = parser.read(nrows)
    455     finally:
    456         parser.close()

~/opt/anaconda3/lib/python3.7/site-packages/pandas/io/parsers.py in read(self, nrows)
   1130 
   1131     def read(self, nrows=None):
-> 1132         nrows = _validate_integer("nrows", nrows)
   1133         ret = self._engine.read(nrows)
   1134 

~/opt/anaconda3/lib/python3.7/site-packages/pandas/io/parsers.py in _validate_integer(name, val, min_val)
    389             val = int(val)
    390         elif not (is_integer(val) and val >= min_val):
--> 391             raise ValueError(msg)
    392 
    393     return val

ValueError: 'nrows' must be an integer >=0
```

**Comment by rahulbhadani** [comment]
Fixed in commit  b80fbb2. Added a check to see if the file is empty, then return with `success` attribute set to False.

**Comment by rahulbhadani** [comment]
2020-09-17-14-07-51_2T3Y1RFV8KC014025_CAN_Messages.csv was an empty file.

```
folder = "/home/ivory/CyverseData/JmscslgroupData/PandaData/2020_09_17/"
file= "2020-09-17-14-07-51_2T3Y1RFV8KC014025_CAN_Messages.csv"
r =strymread(csvfile=folder + file,  verbose=True)

if r.success:
    # then proceed normally

```
